pkg/clockcache/cache.go

```diff
@@ -288,6 +299,20 @@ func (self *Cache) Len() int {
 	return len(self.storage)
 }
 
+func (self *Cache) Evictions() uint64 {
```

Comment:
Wouldn't we have the rate calculation functionality for evictions as per plans? (I was expecting some background goroutine to be doing this job somewhere here)

Reply:
it is but  this loop does not belong in the datastructure itself but in a wrapper